### PR TITLE
Add initial client version tracking

### PR DESCRIPTION
**Pull request by Bruception**
### Description

- Adds client version tracking
Screen Shot 2022-03-16 at 11 00 30 AM

### CODE REVIEW

File: backend/utils/prometheus.ts [hunk 1]
```diff
@@ -144,3 +144,13 @@ export function incrementResult(
 
   resultDuration.observe(res.testDuration);
 }
+
+const clientVersionsCounter = new Counter({
```
Comment (Bruception) [inline backend/utils/prometheus.ts:148]: Not sure what kind of metrics we want to record here.

File: frontend/src/scripts/version.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export const CLIENT_VERSION = "1.0.0";
```
Comment (Bruception) [inline frontend/src/scripts/version.ts:1]: Need to look into whether we can automatically bump this when a PR merges. Each PR would need to specify whether it is a `major/minor/patch` change.

Comment (Miodec) [reply frontend/src/scripts/version.ts:1]: I was thinking to just make the version a timestamp of when the code was deployed

File: backend/utils/prometheus.ts (hunk 1 above)
Comment (Miodec) [reply backend/utils/prometheus.ts:148]: This is good

**Review by Miodec — APPROVED** [review]

**Pull request merged**